contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/gstreamer/category.pxu

```diff
@@ -0,0 +1,3 @@
+unit: category
+id: gstreamer
+_name: Gstreamer tests
```

Comment:
Please also do corresponding change to this _name field

Reply:
Update as 
```
unit: category
id: video-codec
_name: Video Codec tests
```